.github/workflows/ai-lab-e2e-nightly-windows.yaml

```diff
@@ -67,6 +67,11 @@ on:
         description: 'Env. Variables passed into target machine, ie: VAR1=xxx,VAR2=true,VAR3=15,VAR4="Pass me along"'
         type: 'string'
         required: true
+      azure_vm_size:
+        default: 'Standard_D8s_v4'
+        description: 'Azure VM size'
+        type: 'string'
```

Comment:
Maybe worth an enumeration ?

Reply:
Might be worth using options. +1

Reply:
Implemented, added cheaper and more performant machine as default